demo/index.html

```diff
@@ -111,8 +110,8 @@
 
       <paper-dialog id="dropdownDialog">
         <h2>Dialog Title</h2>
-        <paper-dropdown-menu label="Value">
-          <paper-menu class="dropdown-content">
+        <paper-dropdown-menu-light label="Value">
+          <paper-listbox class="dropdown-content" slot="dropdown-content">
```

Comment:
If `class="dropdown-content"` isn't doing anything here anymore, we should remove it.